test/test_checking.cpp

```diff
@@ -63,7 +96,7 @@ enum
 	force_recheck = 8,
 };
 
-void test_checking(int flags = read_only_files)
+void test_checking(int flags = 0)
```

Comment:
I don't see this change affecting anything here. is it left-over from a previous iteration?

Reply:
Well, default value seemed incorrect to me. I was pretty sure that "checking" and "read_only" tests must be different. Before this change they were doing exactly the same.

Reply:
TORRENT_TEST(checking)
{
	test_checking();
}

TORRENT_TEST(read_only)
{
	test_checking(read_only_files);
}

```test_checking() == test_checking(read_only_files)```

Reply:
ah, I see. maybe the default value should be removed even, if it's only relied on in one place

Reply:
I like the idea to remove default argument from ‘test_checking’.